patches/aspnetcore/0004-Match-new-NuGet-MSBuild-version.patch

```diff
@@ -1,29 +0,0 @@
-From 720b538e1aa3a456ed5d518c4b1fa011a7e74e6c Mon Sep 17 00:00:00 2001
-From: Chris Rummel <crummel@microsoft.com>
-Date: Thu, 15 Aug 2019 16:38:08 -0500
-Subject: [PATCH 4/6] Match new NuGet MSBuild version.
-
----
- eng/tools/RepoTasks/RepoTasks.csproj | 6 +++---
- 1 file changed, 3 insertions(+), 3 deletions(-)
-
-diff --git a/eng/tools/RepoTasks/RepoTasks.csproj b/eng/tools/RepoTasks/RepoTasks.csproj
-index 0bf6be98ae..3f8c247666 100644
---- a/eng/tools/RepoTasks/RepoTasks.csproj
-+++ b/eng/tools/RepoTasks/RepoTasks.csproj
-@@ -17,9 +17,9 @@
-   </ItemGroup>
- 
-   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
--    <PackageReference Include="Microsoft.Build.Framework" Version="15.8.166" />
--    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.8.166" />
--    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.8.166" />
-+    <PackageReference Include="Microsoft.Build.Framework" Version="15.7.179" />
```

Comment:
Removal of this will cause these to come back as prebuilts.  I see this version moved up to 16.3.0.  I wonder if that was out of necessity or if they just moved?

Reply:
I will try reverting this to 15.7.179 again and try to track down why they did it.